ddp-workspace/projects/ddp-sdk/src/lib/components/address/addressEmbedded.component.ts

```diff
@@ -189,17 +201,19 @@ export class AddressEmbeddedComponent implements OnDestroy, OnInit {
     componentBusy = new EventEmitter<boolean>(true);
 
     @ViewChild(AddressInputComponent, {static: true}) addressInputComponent: AddressInputComponent;
-    public formErrorMessages$: Observable<string[]>;
+    public formErrorMessages$: Observable<FormError[]>;
     public errorMessagesToDisplay$: Observable<string[]>;
     public suggestionForm: FormGroup;
+    public ignoreEasyPostErrorsCheckbox = new FormControl();
     public isInputComponentBusy$ = new BehaviorSubject<boolean>(false);
     // variables for template
     public suggestionInfo$: Observable<AddressSuggestion | null>;
-    public inputAddress$: Subject<Address | null> = new BehaviorSubject(null);
-    public verifyFieldErrors$: Observable<AddressError[]>;
+    public inputAddress$ = new BehaviorSubject<Address | null>(null);
+    public verifyFieldErrors$: Observable<FieldError[]>;
```

Comment:
just refactoring for these fields

Reply:
I think can stay as `AddressError[]`

Reply:
well, fields errors may contain easyPost error, and we use this data. So we need FieldError interface here (with isEasyPostError property)